openedx/core/djangoapps/content/course_overviews/tests/test_course_overviews.py

```diff
@@ -1,19 +1,23 @@
 """
 Tests for course_overviews app.
 """
-from cStringIO import StringIO
+from __future__ import absolute_import
+
 import datetime
-import ddt
 import itertools
 import math
+from cStringIO import StringIO
```

Comment:
Hmm, taking note of the fact that python-modernize doesn't deal with cStringIO.  We'll probably have to go back and clean that up separately later.